### PR TITLE
Testing memmove function

### PR DESCRIPTION
**Pull request by kouamdo**
Test of memmove function

### CODE REVIEW

File: src/tests/stdlib/memmove_test.c [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * @date June 29, 2020
+ * @author: KOUAM Ledoux
+ */
+
+#include <string.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("memmove test suit");
+
+TEST_CASE("move full data")
```
Comment (anton-bondarev) [inline src/tests/stdlib/memmove_test.c:13]: please use code-style brackets 
Must be 
```
TEST_CASE("move full data") {
```

File: src/tests/stdlib/memmove_test.c [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    test_assert_zero(strcmp(dest,src));
+}
+
+TEST_CASE("Move array with null value in the middle")
+{
+    char src[] = {'s','t','r','\0','i','n','g','\0'};
+    char dest[strlen(src)+1];
+
+     test_assert_equal(strlen(src), \
+    strlen(memmove(dest,src,strlen(src))));
```
Comment (anton-bondarev) [inline src/tests/stdlib/memmove_test.c:49]: Please use tabs (not spaces) at the start of a line (for the whole file)

**Comment by anton-bondarev** [comment]
@kouamdo Thanks!
I've made some comments. Please fix it.
After that, I''' ask you to add your tests in x86/test/unit template with a separate commit
And at the finish stage, I'll ask you to rebase under embox/master and squash some your commits. But now don't rebase, just push your new commits under your branch

File: src/tests/stdlib/memmove_test.c (hunk 1 above)
Comment (kouamdo) [reply src/tests/stdlib/memmove_test.c:13]: Okay , ...

**Comment by alexkalmuk** [comment]
Build is broken - https://github.com/embox/embox/runs/822669418

**Comment by alexkalmuk** [comment]
Build is fixed in https://github.com/embox/embox/pull/2077

BTW, the test fails with 4/5 failures, so be more careful next time, please.

**Comment by alexkalmuk** [comment]
Nevertheless, thank you for this contribution anyway! I hope the test will be fixed in the future and everything will be fine.

**Comment by kouamdo** [comment]
Thank you very much ... i'm so proud🤗🤗

Le mer. 1 juil. 2020 à 09:09, Alex Kalmuk <notifications@github.com> a
écrit :

**Pull request merged**